scripts/README.md

```diff
@@ -21,18 +21,27 @@ understand how the BringAuto software and hardware platform works.
 ### third_party_monitoring
 
 - it helps to observe messages sent by BringAuto Daemon
-- run the script by `python3 third_party_monitoring.py`
 - it may need recompiled protobuf files with following commands:
 - - `git submodule update --init --recursive`
 - - `protoc -I"$(pwd)/../autonomy-host-protocol/" --python_out=. "$(pwd)/../autonomy-host-protocol/CarStateProtocol.proto"`
 - - `protoc -I"$(pwd)/../autonomy-host-protocol/" --python_out=. "$(pwd)/../autonomy-host-protocol/IndustrialPortalProtocol.proto"`
+- run the script by `python3 third_party_monitoring.py`
 
 ### fleet-init
 
 - initializes the fleet management database with stops and routes
-- run the script by `python3 fleet-init/main.py -c init_config.ini -m maps -d`
 - is a submodule so it needs to be initialized by:
 - - `git submodule update --init --recursive`
+- run the script by `python3 fleet-init/main.py -c init_config.ini -m maps -d`
+
+### create_docker_compose_for_testing
+
+- creates a new `docker-compose.yml` with docker images replaced by desired images built from local repositories
+- the config file `docker_compose_for_testing.json` needs to be adjusted
+- - `etna_path` needs to point to the root directory of the etna repository
+- - to build a docker image from a local repository, change the `replace` tag to `true` and set the `path` to the root directory of that repository
+- - the `force_rebuild` tag forces the docker image of that component to be rebuilt
+- run the script by `python3 create_docker_compose_for_testing.py`
 
 ## Arguments
```

Comment:
I recommend removing the separate "Arguments" section and instead adding a small subtitle "Arguments" within each script's description. This will help streamline the documentation and make it less confusing, as having the arguments separated from their respective scripts can be slightly disorienting. However, this is just my personal opinion.

Reply:
Separated arguments into individual script sections